api-gateway/src/main/resources/logback.xml

```diff
@@ -5,7 +5,7 @@
         </encoder>
     </appender>
 
-    <root level="info">
+    <root level="DEBUG">
```

Comment:
better leave debug?

Reply:
I guess debug logging is fine as the product is still in development mode. In this case, it might be easier to trace errors on prod

Reply:
actually, I forgot to rollback this change.
Debug is useful, we have enable debug logging for `logging.level.org.springframework` in backend, demo and orchestrator.
Will do in the same way